projects/schematics/README.md

```diff
@@ -68,94 +68,91 @@ Here are some examples how the `add-cms-component` schematic can be used:
 - `ng g @spartacus/schematics:add-cms-component myAwesomeCms --cms-model=MyModel --module=app` - generates _my-awesome-cms.component.ts_ component, _my-awesome-cms.module.ts_ module and imports it to the specified _app.module.ts_
 - `ng g @spartacus/schematics:add-cms-component myAwesomeCms --cms-model=MyModel --module=app --declareCmsModule=my-cms-path/my-cms` - generates _my-awesome-cms.component.ts_ component and adds it to the specified _my-cms-path/my-cms.module.ts_ module. It also imports _my-cms.module.ts_ to the specified _app.module.ts_
 
-## Building and using Spartacus Schematics from source
-
-This section is for Spartacus developers and anybody else who works with Spartacus source code.
-
-### Prerequisites
+## Prerequisites
 
 Install angular schematics globally: `npm install -g @angular-devkit/schematics-cli`
 
-### Building and testing schematics
+## Testing schematics
 
-1. To build schematics use `yarn build`
-2. To run tests use `yarn test`
+### Unit testing
 
-### Running schematics on separate / new project
+To run schematics unit tests:
 
-1. Run `npm pack` in schematics directory. It will generate the `spartacus-schematics-x.x.x.tgz` file.
-2. Generate a new Angular app (using `ng new` command) or choose an existing one
-3. Install and run schematics in your app using either:
+1. `$ cd projects/schematics`
+2. `yarn test`
 
-- `ng add path-to-file/spartacus-schematics-x.x.x.tgz` (it will execute default schematics)
-- `yarn add path-to-file/spartacus-schematics-x.x.x.tgz` and `ng g @spartacus/schematics:add-spartacus`
+### Integration testing
 
-## Developing update schematics
-
-### The update schematic structure
-
-The `projects/schematics/src/migrations/migrations.json` file contains all migrations for all Spartacus versions:
-
-- _name_ property is important for developers to quickly understand what the migration script is doing. By convention, the migration _name_ should follow `migration-v<version>-<migration-feature-name>-<sequence-number>` pattern, where:
-  - _version_ should indicate for which Spartacus version the migration is intended.
-  - _migration-feature-name_ is a short name that describes what the migration is doing.
-  - _sequence-number_ is the sequence number in which the migrations should be executed
-  - An example is _migration-v2-update-cms-component-state-02_.
-- _version_ is _really_ important for the Angular's update mechanism, as it is used to automatically execute the required migration scripts for the current project's version. For more information about this, please check [releasing update schematics](#releasing-update-schematics) section.
-- _factory_ - points to the specific migration script.
-- _description_ - a short free-form description field for developers.
+The best way to test an unpublished schematic is to publish it to a local npm registry. For more, see [developing update schematics](#Developing-update-schematics)
 
-### Testing update schematic
+## Developing update schematics
```

Comment:
```suggestion
## Developing update for schematics
```

```suggestion
## Developing schematics update
```

Maybe this way? What do you think?

Reply:
After an offline discussion, we decided to keep it as is